packages/moonstone/styles/colors-light.less

```diff
@@ -15,10 +15,10 @@
 @moon-white:       #fff;
 @moon-red:         #f00;
 
-@moon-remote-button-red-color:     #ff6d78; // #da3c3c; // #e71b1c;
-@moon-remote-button-green-color:   #6ff467; // #21c171; // #339966;
-@moon-remote-button-yellow-color:  #ffc600; // #dac139; // #fddb24;
-@moon-remote-button-blue-color:    #0084ff; // #5677d0; // #4764b4;
+@moon-remote-button-red-color:     #e1777e; // #ff6d78; // #da3c3c; // #e71b1c;
```

Comment:
Do we need a complete history of "moonstone red" in source? :)

Reply:
I thought it was nice, it's small, and shows some evolution over time. It didn't hurt anybody and is pruned out during compile. Just a nice little easter egg, especially if you have an editor set up to display hex colors as colors.